### PR TITLE
Updating `<meta name="theme-color" />` 🎨 

### PR DESCRIPTION
**Pull request by sanjaiyan-dev**
Updating `<meta name="theme-color" />` for the website. For more info https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color

meta theme colour-: `#17a2b8`

_Sorry if I made any mistakes :(_

### CODE REVIEW

**Comment by domesticmouse** [comment]
@sanjaiyan-dev: I'm curious where you got that color hexcode from?

**Comment by sanjaiyan-dev** [comment]
> @sanjaiyan-dev: I'm curious where you got that color hexcode from?

@domesticmouse I got it from the CSS of flutter website.

Under `var(--info)` CSS var. And it looks similar to colour in flutter logo too.

**Comment by domesticmouse** [comment]
PTAL @khanhnwin

**Comment by kevmoo** [comment]
Cool, @sanjaiyan-dev !

**Comment by sanjaiyan-dev** [comment]
> Cool, @sanjaiyan-dev !

Thanks 🙌

File: src/_layouts/base.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <link rel="icon" href="/assets/images/shared/brand/flutter/icon/64.png" />
     <link rel="apple-touch-icon" href="/assets/images/shared/brand/flutter/logo/flutter-logomark-320px.png" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <meta name="theme-color" content="#ffffff" />
+    <meta name="theme-color" content="#17a2b8" />
```
Comment (atsansone) [inline src/_layouts/base.html:12]: **issue:** I believe this should be the same color as the link color:
```suggestion
    <meta name="theme-color" content="#0175C2" />
```

If it's the same as the background of callouts, that would be `#2372a3`.

@kevmoo , any thoughts?

Comment (kevmoo) [reply src/_layouts/base.html:12]: Maybe? I'd love to see a screenshot of the output on mobile for context...

**Comment by sfshaza2** [comment]
This PR makes me nervous as it changes the color for the website. @parlough, can you weigh in?

**Comment by parlough** [comment]
I think with the current white theme and white navbar of the site, it makes sense to stick with the current theme color of white. 

Thanks for the PR though, definitely something useful to consider!

**Comment by sfshaza2** [comment]
That makes sense to me. Thanks, @parlough. Closing.

**Pull request closed without merging**